src/Card/CardContext.jsx

```diff
@@ -1,25 +1,28 @@
 import React, { createContext } from 'react';
-import PropTypes from 'prop-types';
+import PropTypes, { bool } from 'prop-types';
 
 const CardContext = createContext({});
 
 const CardContextProvider = ({
-  orientation, children,
+  orientation, children, isLoading,
 }) => (
-  <CardContext.Provider value={{ orientation }}>
+  <CardContext.Provider value={{ orientation, isLoading }}>
     {children}
   </CardContext.Provider>
 );
 
 CardContextProvider.propTypes = {
   /** Specifies which orientation to use. */
   orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+  /** Specifies loading state. */
+  isLoading: bool,
```

Comment:
Why don't you use an approach like you use below `isLoading: PropTypes.bool,`?

Reply:
You're right, thanks